charts/k8s/templates/etcd-statefulset.yaml

```diff
@@ -73,6 +73,11 @@ spec:
         - name: data
           emptyDir: {}
       {{- end }}
+      {{- if .Values.init.manifests }}
```

Comment:
Please add this to the syncer deployment as etcd cannot do anything with the manifests